libbebop/bebop.cc

```diff
@@ -235,6 +236,36 @@ Bebop::doFlatTrimCalibration()
     m_FlatTrimSemaphore.wait();
 }
 
+//! Move to a new pose relative to the current one
+void
+Bebop::relativeMove(meter_t dx, meter_t dy, meter_t dz, radian_t dpsi)
```

Comment:
What's a ``dpsi`` angle?

Reply:
Change in yaw -- it's what the Bebop API calls it. ``yaw`` would probably be a better name for this though.